cmd/migrations/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BUILDER=koor-tech/genesis-base-builder:latest
+FROM $BUILDER as builder
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o migrate cmd/migrations/main.go
+
+
+FROM alpine:latest
+RUN apk add --no-cache bash
+
+COPY --from=builder /app/migrate /genesis/bin/migrate
+COPY cmd/migrations/migrations /genesis/bin/migrations
+ENTRYPOINT ["/genesis/bin/migrate", "migrate", "up"]
```

Comment:
Is there a reason why we don't just build one "big" image with our 2-3 genesis related binaries?
A few more megabytes to download don't hurt and having the migrations in the same image wouldn't hurt, right?

Reply:
I was doing some tests using the same image but I think is better to keep them separately since they are for different tasks and I think is better for example deploy the migration in a sidecar or init container and remove it after it finished like migrations container, by other side is better to keep the worker separately since we need to keep ir running all the time and that is pretty easy to manage by k8s.

Also I created this containers from sratch so they are pretty small 73 and 12 MB respectively 
```bash
REPOSITORY                                                  TAG            IMAGE ID       CREATED              SIZE
koor-tech/genesis-worker                                    latest         ead7c7bcc7ff   About a minute ago   73.3MB
koor-tech/genesis-base-builder                              latest         29c859bcf80e   2 minutes ago        816MB
koor-tech/genesis-migrate                                   latest         bda4e2ead770   20 minutes ago       12.2MB
```

Does it make sense?

Reply:
It depends on the view and how we deploy, but let's go ahead with it.